samples/excel/30-events/events-comment-event-handler.yaml

```
@@ -0,0 +1,195 @@
+order: 13
+id: excel-events-comment-event-handler
+name: Comment events
+description: 'Registers event handlers to listen for comment additions, changes, and deletions.'
+host: EXCEL
+api_set:
+    ExcelAPI: '1.12'
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup)); 
+        $("#registerEventHandlers").click(() => tryCatch(registerEventHandlers)); 
+        $("#add-comment-to-selected-cell").click(() => tryCatch(addCommentToSelectedCell)); 
+        $("#add-comment-to-cell").click(() => tryCatch(addCommentToCell)); 
+        $("#edit-comment").click(() => tryCatch(editComment)); 
+        $("#delete-comment").click(() => tryCatch(deleteComment));
+
+        async function registerEventHandlers() {
+            await Excel.run(async (context) => {
+                const comments = context.workbook.worksheets.getActiveWorksheet().comments;
+
+                // Register the onAdded, onChanged, and onDeleted comment event handlers.
+                comments.onAdded.add(commentAdded);
+                comments.onChanged.add(commentChanged);
+                comments.onDeleted.add(commentDeleted);
+
+                await context.sync();
+
+                console.log("Added handlers for Comment onAdded, onChanged, and onDeleted events.");
+            });
+        }
+
+        async function commentAdded(event: Excel.CommentAddedEventArgs) {
+            await Excel.run(async (context) => {
+                // Retrieve the added comment.
```

Consider pointing out that the ID is what is being used to map the event to the comment.
```suggestion
                // Retrieve the added comment using the comment ID from the event args.
```

(Note that if you make this or a related change, you should adjust the comment in the other event listeners.)

---

I implemented part of this. Thanks for the suggestion!